lib/parsers/aac_parser.rb

```diff
@@ -0,0 +1,34 @@
+require_relative 'aac_parser/adts_header_info'
+class FormatParser::AACParser
+  include FormatParser::IOUtils
+
+  AAC_MIME_TYPE = 'audio/aac'
+
+  def likely_match?(filename)
+    filename =~ /\.aac?$/i
```

Comment:
qq: Are .aa files also AAC?

Reply:
No, .aa files aren't AAC. I will update the regex.

Reply:
Updated.